basil/TL/SensirionSensorBridge.py

```diff
@@ -0,0 +1,78 @@
+#
+# ------------------------------------------------------------
+# Copyright (c) All rights reserved
+# SiLab, Institute of Physics, University of Bonn
+# ------------------------------------------------------------
+#
+
+import logging
+import array
+
+from basil.TL.TransferLayer import TransferLayer
+
+logger = logging.getLogger(__name__)
+
+from sensirion_shdlc_driver import ShdlcSerialPort, ShdlcConnection
+from sensirion_shdlc_sensorbridge import SensorBridgePort, SensorBridgeShdlcDevice
+from sensirion_shdlc_sensorbridge.device_errors import SensorBridgeI2cTimeoutError
```

Comment:
You could try to catch import error and give a link to install.

Reply:
its on pypi. https://pypi.org/project/sensirion-shdlc-driver/
should be a prerequisit to basil and added to `setup.py`?

Reply:
Probably not since most people will not sue it. Not very strong opinion.

Reply:
Should not be in setup.p; otherwise we clutter it too much with dependencies.